tuna/parse_args.py

```diff
@@ -164,3 +165,67 @@ def args_check(args: argparse.Namespace, parser: argparse.Namespace) -> None:
     parser.error(
         "When setting up a new tunning session the following must be specified: "\
         "label.")
+
+
+@typing.no_type_check
+def get_import_cfg_parser(
+    with_yaml: bool = True) -> jsonargparse.ArgumentParser:
+  """Return parser for import_configs subcommand"""
+
+  parser = setup_arg_parser(
+      'Import MIOpenDriver commands and MIOpen performance DB entries.',
```

Comment:
Change info string to generic message like 'Import configurations for module'
This message can be overwritten in get_import_cfg_parser_miopen etc.